test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_InvalidValueType.lua

```diff
@@ -0,0 +1,52 @@
+---Verification Criteria
```

Comment:
@Aleksandriys, replace you "Verification criteria" with requirement summary+description+expected result block due to template